src/EmergencyContent.elm

```diff
@@ -217,3 +218,16 @@ iconStyle =
         [ height (px 50)
         , marginTop (px 10)
         ]
+
+
+linkStyle : Style
+linkStyle =
+    batch
+        [ color purple
+        , hover
+            [ color darkOrange ]
+        , focus
+            [ border3 (px 3) solid teal.colour
+            , outline zero
```

Comment:
I think outline works better for inline elements where they wrap. We've used border elsewhere because it  responds to the border radius on our curved buttons, where outline doesn't.

Reply:
So do I just remove the focus state and it will default to something better?